lib/app/config/home_screen.dart

```diff
@@ -0,0 +1,40 @@
+// Stateful navigation based on:
+// https://github.com/flutter/packages/blob/main/packages/go_router/example/lib/stateful_shell_route.dart
+import 'package:black_tax_and_white_benefits/app/config/scaffold_navigation_bar.dart';
+import 'package:black_tax_and_white_benefits/app/config/scaffold_navigation_rail.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({
+    Key? key,
+    required this.navigationShell,
+  }) : super(key: key ?? const ValueKey<String>('ScaffoldWithNestedNavigation'));
+  final StatefulNavigationShell navigationShell;
```

Comment:
i hadn't heard of navigationShell, that is interesting

Reply:
https://codewithandrea.com/articles/flutter-bottom-navigation-bar-nested-routes-gorouter/